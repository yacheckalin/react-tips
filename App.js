import React from "react";

import SearchForm from "./react-patterns/controlled-component/SearchForm";
import Clicker from "./react-patterns/state-hoisting/Clicker";
import NavigationLayout from "./react-patterns/layout-component/NavigationLayout";
import UserContainer from "./react-patterns/container-component/UserContainer";
import Message from "./react-patterns/render-props/Message";

const App = () => (
  <div>
    <SearchForm />
    <Clicker />
    <NavigationLayout />
    <UserContainer />
    <Message />
  </div>
);

export default App;
