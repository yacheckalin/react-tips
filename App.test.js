import App from "./App";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import React from "react";

describe("<App />", () => {
  it("renders and displays properly", () => {
    const wrapper = shallow(<App />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
