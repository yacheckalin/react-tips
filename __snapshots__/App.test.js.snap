// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders and displays properly 1`] = `
<div>
  <SearchForm />
  <Clicker />
  <NavigationLayout />
  <UserContainer />
</div>
`;
